src/get_maintainer_wrapper.sh

```diff
@@ -1,90 +1,240 @@
 . $src_script_path/kwio.sh --source-only
 
-
+# Prints the authors of a given file or files inside a given dir.
+#
+# @parameter The argument is a file or directory path
```

Comment:
How about use "@FILE_OR_DIR The ..." instead of @parameter? The Kernel documentation use this style.